openstack/pipeline/02configure

```diff
@@ -635,6 +635,29 @@ do
             assert_min_release queens 'cinder-lvm'
             MOD_OVERLAYS+=( "cinder-lvm.yaml" )
             ;;
+        --cinder-volume)
+            assert_min_release ussuri 'cinder-volume'
+            MOD_OVERLAYS+=( "cinder-volume.yaml" )
+            MOD_PARAMS[__CINDER_VOLUME_INTERFACE__]='cinder-volume'
+            ;;
+        --cinder-purestorage)
+            assert_min_release ussuri 'cinder-volume'
+            MOD_OVERLAYS+=( "cinder-purestorage.yaml" )
+            MOD_MSGS[cinder-purestorage.0]="For access to the serverstack purestorage SAN please contact the OpenStack team for the API Token / IP and specify them using --purestorage-api-token and --purestorage-san-ip"
+            MOD_MSGS[cinder-purestorage.1]="Volumes created on the PureStorage SAN are not easily identifiable - you cannot see which user created them and they all have the name volume-%s. Please ensure you delete all volumes before destroying this environment to prevent orphaned volumes accumulating."
+            MOD_MSGS[cinder-purestorage.2]="After creation create volume types like this:"
+            MOD_MSGS[cinder-purestorage.3]="openstack volume type create cinder-purestorage --property volume_backend_name=cinder-purestorage"
+            MOD_MSGS[cinder-purestorage.4]="openstack volume type create cinder-ceph --property volume_backend_name=cinder-ceph # If also deploying ceph"
+            MOD_MSGS[cinder-purestorage.5]="juju config cinder default-volume-type=cinder-ceph # or cinder-purestorage"
+            ;;
+        --purestorage-api-token)  #__OPT__type:<str>
+            MOD_PARAMS[__PURESTORAGE_API_TOKEN__]=$2
+            shift
```

Comment:
If i omit these option i get an error like:

ERROR: there are unrendered variables in your bundle:
./b/o/cinder-purestorage.yaml:      pure-api-token: __PURESTORAGE_API_TOKEN__
./b/o/cinder-purestorage.yaml:      san-ip: __PURESTORAGE_SAN_IP__

Another way to solve this is to use get_param e.g.

```
        msg="REQUIRED: purestorage api token"
        get_param $1 __PURESTORAGE_API_TOKEN__ "$msg"
        msg="REQUIRED: purestorage san ip:"
        get_param $1 __PURESTORAGE_SAN_IP__ "$msg"
```

Reply:
This didn't let you still use --purestorage-san-ip as well (it seems?) and you'd have to type them in manually every time. So I just added MOD_DEFAULTS instead so it renders them blank but still has the MOD_MSGS warning. Let me know if that works for you.